django_logging/logger.py

```diff
@@ -7,20 +7,23 @@
 from . import settings
 
 LOG_LEVEL = settings.LOG_LEVEL.upper()
-LOG_HANDLERS = ['default']
+LOG_HANDLERS = []
```

Comment:
If the settings.*_LOG are all set to `false`, it means that nothing will get logged anymore, not sure if this is desirable?

Reply:
Yes, but all settings must be explicitly set to false. 
If you prefer, it is possible to raise an exception in this case!

Reply:
Couldn't the option to silence the log be interesting?

Reply:
I guess the side effect can actually be considered a feature :)